core/domain/prod_validation_jobs_one_off.py

```diff
@@ -92,7 +93,28 @@ def map(model_instance):
         if not model_instance.deleted:
             model_name = model_instance.__class__.__name__
             validator_cls_name = '%sValidator' % model_name
-            validator = getattr(prod_validators, validator_cls_name)
+            # Module name for models is of the form:
+            # 'core.storgae.<model-type>.gae_models'.
+            # Module name for validators is of the form:
+            # 'core.domain.<model-type>'.
```

Comment:
is it `'core.domain.<model-type>_validators'`?

Reply:
Done.